src/clusterfuzz/_internal/google_cloud_utils/storage.py

```diff
@@ -1170,3 +1170,30 @@ def get_signed_download_url(remote_path, minutes=SIGNED_URL_EXPIRATION_MINUTES):
   contents."""
   provider = _provider()
   return provider.sign_download_url(remote_path, minutes=minutes)
+
+
+def get_blob_signed_upload_url():
+  """Returns a pair of (blob_name,signed_upload_url) to be used from utask_main
+  to upload blobs."""
+  bucket = blobs_bucket()
+  from clusterfuzz._internal.google_cloud_utils import blobs
```

Comment:
Don't do this please it introduces a circular import. Can you put this in blobs.py?

Reply:
Done.